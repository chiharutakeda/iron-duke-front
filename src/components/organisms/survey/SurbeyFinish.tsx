import React from 'react';
import { useRouter } from 'next/router';
const SurbeyFinish = () => {
  const router = useRouter();
  return (
    <>
      <div>SurbeyFinish</div>
      <button
        onClick={() => {
          router.push('SurbeyFirst');
        }}
      >
        次へ
      </button>
    </>
  );
};

export default SurbeyFinish;
