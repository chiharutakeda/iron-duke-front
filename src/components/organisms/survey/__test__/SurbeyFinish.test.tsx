import ReactTestRenderer from 'react-test-renderer';
import SurbeyFinish from '../SurbeyFinish';

it('SurbeyFinish', async () => {
  const tree = await ReactTestRenderer.create(
    <SurbeyFinish />
  ).toJSON();
  await expect(tree).toMatchSnapshot();
});
