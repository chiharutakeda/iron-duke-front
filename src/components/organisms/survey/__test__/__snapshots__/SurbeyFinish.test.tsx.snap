// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SurbeyFinish 1`] = `
Array [
  <div>
    SurbeyFinish
  </div>,
  <button
    onClick={[Function]}
  >
    次へ
  </button>,
]
`;
